test_regress/driver.pl

```diff
@@ -849,7 +848,8 @@ sub compile_vlt_flags {
     unshift @verilator_flags, "--trace" if $opt_trace;
     my $threads = ::calc_threads($Vltmt_threads);
     unshift @verilator_flags, "--threads $threads" if $param{vltmt} && $checkflags !~ /-threads /;
-    unshift @verilator_flags, "--trace-fst-thread" if $param{vltmt} && $checkflags =~ /-trace-fst/;
+    unshift @verilator_flags, "--trace-threads 1" if $param{vltmt} && $checkflags =~ /-trace /;
+    unshift @verilator_flags, "--trace-threads 2" if $param{vltmt} && $checkflags =~ /-trace-fst /;
```

Comment:
Maybe this is the answer to the testing question?

Reply:
Yes, this seems the right sort of defaulting.  However note Travis-CI has only 2 CPUs so I think this will result in asking for 4 threads which might make it run slow - see if the test time of post change on travis goes up.

Reply:
Just FYI, Travis takes the same time within normal variation.